stellar_rust_sdk/src/models/mod.rs

```diff
@@ -6,6 +6,49 @@ pub mod prelude {
     pub use super::Response;
 }
 
+pub trait PostRequest {
```

Comment:
Moeten we deze wellicht limiteren tot `pub crate`? Eigenlijk zou dat dan ook moeten voor de `request` struct. Idealiter gaan consumers van deze crate hier niet mee aan de haal.

Reply:
Ja, goed punt. `Request` gelijk meenemen?

Reply:
Yes, laten we dat doen